lensless/benchmark.py

```diff
@@ -170,13 +240,28 @@ def benchmark(model, dataset, batchsize=1, metrics=None, **kwargs):
     assert isinstance(model._psf, torch.Tensor), "model need to be constructed with torch support"
     device = model._psf.device
 
+    def Reconstruction_Error(prediction, lensless):
```

Comment:
lower case for function -> `reconstruction_error`
and add docstring to know what's being computed

Reply:
As of now, this isn't an independent function. I don't think this need to be documented with a docstring. A simple comment should be enough.
Or, I can make it a proper function in utils, with its own docstring.

Reply:
as a method of ReconstructionAlgorithm?